### PR TITLE
LPS-177377 Remove Stream usages for module: commerce-service

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java [hunk 1]
```diff
@@ -139,17 +139,11 @@ public List<CommerceShipment> getCommerceShipments(
 			getPermissionChecker(), null,
```
Comment (julschong) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java:139]: @dantewang do you think this _portletResourcePermission.contains() is used incorrectly?
it returns a boolean that is not used unless it has side effect.

**Comment by julschong** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java (hunk 1 above)
Comment (dantewang) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java:139]: @julschong Yes, I think so. Please fix it with a separate ticket and send to Commerce team for review, or report to Commerce team.

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java [hunk 2]
```diff
@@ -242,14 +225,9 @@ public int getCommerceShipmentsCount(long companyId)
...
 
-		Stream<CommerceChannel> stream = commerceChannels.stream();
-
-		long[] commerceChannelGroupIds = stream.mapToLong(
-			CommerceChannel::getGroupId
-		).toArray();
-
 		return commerceShipmentLocalService.getCommerceShipmentsCount(
-			commerceChannelGroupIds);
+			TransformUtil.transformToLongArray(
```
Comment (dantewang) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java:229]: @julschong A lot of methods in this class calls `_commerceChannelService.search(companyId)`, transform the result list to `long[]` of groud ids, and use the array in a local service call.

However, they are inconsistent -- some methods do an `isEmpty` check on the returned list from `search`, but some don't.

Basically, we should avoid unnecessary local service calls wherever possible. Please add shortcut to these methods.

Comment (dantewang) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java:229]: @julschong CommerceSubscriptionEntryServiceImpl also has this issue.

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java (hunk 1 above)
Comment (julschong) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceShipmentServiceImpl.java:139]: @dantewang I sent a PR to commerce team here: https://github.com/liferay-commerce/liferay-portal/pull/3484
this PR will likely need to wait

**Pull request closed without merging**